style.css

```diff
@@ -170,11 +167,11 @@ iframe {
 .video-buttons p{
```

Comment:
You use this selector on line 146. Don't duplicate selectors.
You also have a duplicate `video-buttons` selector on line 163